### PR TITLE
fix: base image for online app

### PR DESCRIPTION
**Pull request by amalucelli**

### CODE REVIEW

File: Dockerfile.build [hunk 1]
```diff
@@ -1,5 +1,4 @@
-# https://github.com/buildkite/agent/blob/master/packaging/docker/ubuntu-linux/Dockerfile
-FROM python:3.6
+FROM pyCarol:2.18.0
```
Comment (rafarui) [inline Dockerfile.build]: I think we should use only 2.20+
I think 2.18 is not organization compatible.

Comment (amalucelli) [reply Dockerfile.build]: I wasn't able to find on the releases: https://hub.docker.com/r/totvslabs/pycarol/tags where is this being pushed?

Comment (rafarui) [reply Dockerfile.build]: hum.. I know it pushes the "latest" for every master merge. it seems it is not creating the tags though... 
@gzamboni , how this works?

Comment (gzamboni) [reply Dockerfile.build]: the tags on the docker hub are created when a tag is pushed to pycarol repository. 
the last tag pushed to pycarol is 2.18
master is tagged on docker hub as "latest".
Let's remember this, It is necessary every time you push a new version to the master branch to remember to tag a new version also.

Comment (gzamboni) [reply Dockerfile.build]: the tags on the docker hub are created when a tag is pushed to pycarol repository.
the last tag pushed to pycarol is 2.18
master is tagged on docker hub as "latest".
Let's remember this, It is necessary every time you push a new version to the master branch to remember to tag a new version also.

Comment (rafarui) [reply Dockerfile.build]: nice.. I will do that

Comment (amalucelli) [reply Dockerfile.build]: @rafarui, can you let me know when it's done, so I can update this PR?

Comment (rafarui) [reply Dockerfile.build]: already created the tags 2.23 and 2.24. 
It might take some minutes to create.

Comment (gzamboni) [reply Dockerfile.build]: Just follow https://hub.docker.com/r/totvslabs/pycarol/tags

Comment (gzamboni) [inline Dockerfile.build]:
```suggestion
FROM pyCarol:2.24.0
```

Comment (amalucelli) [reply Dockerfile.build]: I will do for the other cookie project, but who is responsible to maintain this updated?

**Pull request merged**